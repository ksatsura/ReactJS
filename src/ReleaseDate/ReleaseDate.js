//imports
import React from 'react';
import '../style.css';

const ReleaseDate = (props) => {

    return <p className='release-date'>{props.date}</p>;
};

export default ReleaseDate;