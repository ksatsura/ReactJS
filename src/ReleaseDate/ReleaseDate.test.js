import React from 'react';
import ReleaseDate from './ReleaseDate';
import { shallow } from 'enzyme';

describe('ReleaseDate.js', () => {

    const props = {
        date: '',
    };

    describe('renders', () => {

        it('renders for desktop', () => {
            const component = shallow(<ReleaseDate {...props} />);

            expect(component).toMatchSnapshot();
        });

        it('renders for mobile', () => {
            const component = shallow(<ReleaseDate {...props} isMobile />);

            expect(component).toMatchSnapshot();
        });
    });
});
