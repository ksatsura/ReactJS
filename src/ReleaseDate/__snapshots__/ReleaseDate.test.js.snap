// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseDate.js renders renders for desktop 1`] = `ShallowWrapper {}`;

exports[`ReleaseDate.js renders renders for mobile 1`] = `ShallowWrapper {}`;
